api/swagger.yml

```diff
@@ -594,12 +594,62 @@ components:
         ref:
           type: string
 
+    TaskInfo:
+      type: object
+      required:
+        - id
+      properties:
+        id:
+          type: string
+          description: ID of the task
+
+    RefsDumpStatus:
+      type: object
+      required:
+        - id
+        - completed
+      properties:
+        id:
+          type: string
+          description: ID of the task
+        completed:
+          type: boolean
+        error:
+          type: string
+        refs:
+            $ref: "#/components/schemas/RefsDump"
+
+    RefsRestoreTask:
+      type: object
+      required:
+        - id
+      properties:
+        id:
+          type: string
+          description: ID of the task
+        refs:
+          $ref: "#/components/schemas/RefsDump"
+
+    RefsRestoreStatus:
+      type: object
+      required:
+        - id
+        - completed
+      properties:
+        id:
+          type: string
+          description: ID of the task
+        completed:
+          type: boolean
+        error:
+          type: string
+
     RefsDump:
       type: object
       required:
-        - branches_meta_range_id
-        - tags_meta_range_id
         - commits_meta_range_id
+        - tags_meta_range_id
+        - branches_meta_range_id
```

Comment:
Is the order change significant?

Reply:
nope - it should not have an any effect. also the type name was changed as it is a new operation.